src/kibana/components/config/defaults.js

```diff
@@ -32,6 +32,10 @@ define(function (require) {
       value: null,
       description: 'The index to access if no index is set',
     },
+    'defaultColumns': {
+      value: '["_source"]',
```

Comment:
While it appears type: json has been removed, this value is still being specified as a string?